brigade-worker/src/k8s.ts

```diff
@@ -626,11 +630,16 @@ export class JobRunner implements jobs.JobRunner {
           this.logger.log("Pod not yet scheduled");
           return;
         }
+
         let phase = this.pod.status.phase;
         if (phase == "Succeeded") {
           clearTimers();
           let result = new K8sResult(phase);
           resolve(result);
+        } else if (phase == "Running") { // make sure Pod is running before we start following its logs
```

Comment:
nit: can the comment be on the previous line?
(This way, we're consistent with the single line comments - at least in this file).

Reply:
sure, fixing